care/facility/tests/test_apis/test_ambulance_api.py

```diff
@@ -0,0 +1,285 @@
+from typing import Any
+
+from rest_framework import status
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'rest_framework'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5340830)